### PR TITLE
Show email when completing recurring donation (fixes #376)

### LINKED ISSUE #376: Show email address after submitting a recurring donation
After completing a recurring donation with an email address, the completion page should show that email. Currently it shows nothing because it's looking for the signed in user's email.

Supports https://github.com/mozilla/payments/issues/117

### PR DESCRIPTION
**Pull request by kumar303**
Fixes https://github.com/mozilla/payments-ui/issues/376

### CODE REVIEW

File: public/js/views/transaction/complete-payment.jsx [hunk 1]
```diff
@@ -49,13 +50,18 @@ export default class CompletePayment extends Component {
         <span className="email">{this.props.userEmail}</span>
       </p> : '');
 
+    var product = products.get(this.props.productId);
+    var completionMsg = (product.seller.kind === 'donations' ?
```
Comment (muffinresearch) [inline public/js/views/transaction/complete-payment.jsx]: Nice

**Comment by muffinresearch** [comment]
r+

File: public/js/views/transaction/complete-payment.jsx (hunk 1 above)
Comment (kumar303) [inline public/js/views/transaction/complete-payment.jsx]: I still feel like `donation` should be configured on the product level

Comment (muffinresearch) [inline public/js/views/transaction/complete-payment.jsx]: yeah I agree, but still - I like that the message is specific.

**Pull request merged**